docker-compose.override.yml

```diff
@@ -0,0 +1,41 @@
+version: '3'
+
+services:
+  init:
+    build:
+      context: ./docker/init
+      dockerfile: Dockerfile
+    image: elifesciences/peerscout_init:${IMAGE_TAG}
+    volumes:
+      - config-aws:/home/elife/volume-config-aws
```

Comment:
some weird copying from one folder to the other to avoid ownership clashes?

Reply:
No problem with that, but why needing a volume if the copying is executed every time the container starts anyway?

Reply:
I find that when the container is not run as root or the developer's user, then just mounting the credentials won't work because the permissions are meant to set up that only the user can read them. This makes the developers credentials available via the volume. (The init container is run as root, and therefore has permissions)

One alternative would maybe be to run the container using the developer's user (which seem to also be more complicated than it should). I experimented with a few approaches across the projects. Not sure which one is best. Any suggestions?